application-tour-ui/src/main/resources/TourCode/StepSheet.xml

```diff
@@ -238,6 +239,7 @@ require(['jquery'], function ($) {
                   targetPage  : (targetPage? targetPage: ''),
                   targetAction: (targetAction? targetAction: ''),
                   queryString : (queryString? queryString: ''),
+                  reflex      : (reflex? reflex: 0),
```

Comment:
I can change it, it really shouldn't matter at all. Maybe the person who wrote this originally simply copy-pasted and was a bit lazy like me :)?